frigate/utils/parsers.py

```diff
@@ -209,3 +209,134 @@ def stats_parser_args():
         raise ValueError("No columns provided")
 
     return args
+
+
+def loop_parser():
```

Comment:
there is a lot of overlap with another parser am I right? If so, just call that parser in here, and add the arguments you need maybe?